TTS/tts/utils/text/characters.py

```diff
@@ -28,11 +28,20 @@ def parse_symbols():
 # Phonemes definition (All IPA characters)
 _vowels = "iyɨʉɯuɪʏʊeøɘəɵɤoɛœɜɞʌɔæɐaɶɑɒᵻ"
 _non_pulmonic_consonants = "ʘɓǀɗǃʄǂɠǁʛ"
-_pulmonic_consonants = "pbtdʈɖcɟkɡqɢʔɴŋɲɳnɱmʙrʀⱱɾɽɸβfvθðszʃʒʂʐçʝxɣχʁħʕhɦɬɮʋɹɻjɰlɭʎʟ"
+_pulmonic_consonants = "pbtdʈɖcɟkɡqɢʔɴŋɲɳnɱmʙrʀⱱɾɽɸβfvθðszʃʒʂʐçʝxɣχʁħʕhɦɬɮʋɹɻjɰlɭʎʟt̪"
 _suprasegmentals = "ˈˌːˑ"
 _other_symbols = "ʍwɥʜʢʡɕʑɺɧʲ"
 _diacrilics = "ɚ˞ɫ"
-_phonemes = _vowels + _non_pulmonic_consonants + _pulmonic_consonants + _suprasegmentals + _other_symbols + _diacrilics
+_numbers = "123456789"
```

Comment:
Aren't numbers supposed to be taken care of in the `text_cleaners`?

Reply:
To my understanding, the text_cleaners module is applied right before the phonemizer. So it does not affect the phoneme sequence. These _numbers are just the notion of Vietnamese phonetics, which was produced by espeak-ng.
https://github.com/coqui-ai/TTS/blob/d6ad9a05b40e9f3aa876ecfa1c0d23339f373df4/TTS/tts/utils/text/tokenizer.py#L107

Reply:
@vndee what you mean by "just the notion of Vietnamese phonetics"? Are they more than numbers in Vietnamese?

Reply:
> @vndee what you mean by "just the notion of Vietnamese phonetics"? Are they more than numbers in Vietnamese?
No only 1-9! I think his means is the IPA phonetic of Vietnamese which produced by the espeak-ng including numbers..
ie..'Xin chào các bạn' => s|i|n|1| |c|a|w|2| |k|a|k|5| |b|a|n|6| |. 
That why he add the _numbers.

Reply:
@vndee I think it only contains numbers from 1-6